rust/arrow/src/memory.rs

```diff
@@ -18,80 +18,35 @@
 //! Defines memory-related functions, currently mostly to make this library play nicely
 //! with C.
```

Comment:
is this comment still relevant?

Reply:
This doesn't seem related anymore. Will delete later.